pkg/node/manager.go

```diff
@@ -15,12 +15,28 @@
 package node
 
 import (
+	"bytes"
+	"fmt"
 	"net"
+	"os/exec"
+	"strings"
 	"sync"
 
 	"github.com/cilium/cilium/pkg/maps/tunnel"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/vishvananda/netlink"
+)
+
+// RouteType represents the route type to be configured when adding the node
+// routes
+type RouteType int
+
+const (
+	// BPFRoute is the route type to set up the BPF tunnel maps
+	BPFRoute RouteType = 1 << iota
```

Comment:
`BPFRoute` is a weird name because this has nothing to do with BPF

Reply:
What about `CiliumRoute` or `BPFTunnel`?

Reply:
I've change it to `BPFTunnel`, let me know if that's ok.